### DISCUSSION TITLE
Sanctions for EAA in different countries

### DISCUSSION THREAD

**Discussion by erikgustafsson** [Nordic-Accessibility-Community-Group/en-301-549-resources-and-eaa-monitoring#8] [Ideas]
### Summary
This discussion is for collecting information about sanctions in the EU. Goal is to make a information page in this repo.

#### Ireland 
- Fines not exceeding €60,000 or
- Imprisonment for a term not exceeding 18 months
- Or both 
- - Source: [Lainey Feingold legal update at CAN](https://drive.google.com/drive/folders/1q9ZJirsnN-ixn9SiJa26Y7K1PFOYht0Z).

Prison and fines [European Accessibility Act Implemented into Irish Law](https://www.mhc.ie/latest/insights/european-accessibility-act-implemented-into-irish-law)

#### Germany
[Barrierefreiheitsstärkungsgesetz law](https://bfsg-gesetz.de/37-bfsg/) see paragraph 2

#### Latvia
Latvia includes confiscation and destruction of property as part of the EAA implementation. Quite hefty fines for repeat offenses too! Up to 1 million euros.

- Fines of 500 euros to 1,000,000 euros as a repeat offense
- Courts can order confiscation and destruction of property used in the offense as well as the confiscation of illicit profits (Art. 33)
- - Source: [Lainey Feingold legal update at CAN](https://drive.google.com/drive/folders/1q9ZJirsnN-ixn9SiJa26Y7K1PFOYht0Z).

**Comment by rraddatch** [comment]
## Slovakia

The Slovak transposition of the EAA is **Act No. 351/2022 Coll.** on the accessibility of products and services for persons with disabilities.

### Supervisory Authority
The **Slovak Trade Inspection** (*Slovenská obchodná inšpekcia*) is the body responsible for market surveillance and monitoring compliance with accessibility requirements.

### Sanctions and Administrative Offenses
Section 10 of the Act defines the administrative offenses for service providers and the corresponding fines. Fines range from **€200 to 3% of the turnover** for the previous accounting period. 

The law sets the following **maximum caps** based on the severity of the offense:
*   **€6,000** for failing to meet service accessibility requirements or failing to make accessibility information available to the public.
*   **€20,000** for failing to implement procedures to maintain ongoing compliance, failing to inform the authority of non-compliance, or failing to take corrective actions.
*   **€30,000** for failing to provide the necessary information, documentation, or cooperation to the supervisory authority.

### Specific Sanctions for Multimodal Access Services
For specialized services like electronic program guides (EPG) or hybrid TV, the Act introduces a specific fine ranging from **€100 to €3,000** for non-compliance with multimodal access obligations.

### Exemptions and Fine Waivers
The supervisory authority will **waive the fine** for certain offenses if the provider:
*   Takes the required **corrective measures** within the specified timeframe.
*   Demonstrates that compliance would impose a **disproportionate burden** (*neprimeraná záťaž*).

### Effective Date
The provisions of the Act, including the power to issue sanctions, come into effect on **June 28, 2025**.

**Source:** [Act No. 351/2022 Coll.](https://www.slov-lex.sk/ezbierky/pravne-predpisy/SK/ZZ/2022/351/20260530.html)

**Reply by erikgustafsson** [reply]
Thank you for valuable contributions @rraddatch!

**Comment by 2biazdk** [comment]
Thanks, @rraddatch! We’ve included the Slovakia sanctions information in [PR #41](https://github.com/Nordic-Accessibility-Community-Group/en-301-549-resources-and-eaa-monitoring/pull/41) as part of a new page about EAA sanctions by country. Thank you for contributing this.
